src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -46,7 +46,6 @@ public void Start_StandardInputEncodingPropagatesToStreamWriter()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Retrieving information about local processes is not supported on uap")]
```

Comment:
Not for this PR, but after having removed the netfx runs previously and now the uap ones, many of our test projects are netcoreapp-only... which means we can recombine files like ProcessTests.cs and ProcessTests.netcoreapp.cs.

Reply:
https://github.com/dotnet/corefx/issues/41821